x/epochs/go.mod

```diff
@@ -0,0 +1,145 @@
+module github.com/osmosis-labs/osmosis/x/epochs
```

Comment:
This module name is a bit confusing tbh. When ever we do the import, it is not evident that this is actually a module vs other imports that are not modular.
```
import (
    "github.com/osmosis-labs/osmosis/v11/x/incentives/keeper"
    "github.com/osmosis-labs/osmosis/x/epochs/keeper"
)
```
The only difference being the version after osmosis.

Inoder to avoid confusion for this, I was looking into something like `osmosis.io/x/epochs` as this module, would have to do some more research into packaging and publishing the module. Versioning of this module also would be then easier and not conflicting with other packages.

This would then be in harmony with the efforts on the cosmos-sdk side of things as well.

PS. `osmosis.io/` can use a different name as well.

What do you think? @puneet2019 @ValarDragon @vuong177

Reply:
I have no preference tbh. as long as it works :P

Reply:
@puneet2019 - depending on what you'd like to do you may wish to check out:

https://github.com/eve-network/eve

Also, if you could let us know if epochs can be imported, that'd be super!

Reply:
Hey, we are still figuring this out. 
Seems like tests are a bit of a problem. Short term even I have forked the epochs module using `git subtree` .